src/System.Private.CoreLib/shared/System/ComponentModel/DefaultValueAttribute.cs

```diff
@@ -21,9 +20,8 @@ public class DefaultValueAttribute : Attribute
         /// </devdoc>
         private object _value;
 
-        // We cache reflection types for TypeConverter conversion
-        static Func<MethodInfo> s_getConverterMethodCached;
-        static Func<MethodInfo> s_convertFromInvariantStringMethodCached;
+        // Delegate 'TypeDescriptor.InternalConvertFromInvariantString' reflection object cache
```

Comment:
Nit: I think we can call the method just `ConvertFromInvariantString` and drop the internal from all the names.

Reply:
done